docbook/reference/en/en-US/modules/Maven_and_RESTEasy.xml

```diff
@@ -6,100 +6,87 @@ JBoss's Maven Repository is at:
-      <groupId>org.jboss.resteasy</groupId>
-      <artifactId>resteasy-guice</artifactId>
-      <version>3.1.3-SNAPSHOT</version>
-   </dependency>
+   <para>
 
-   <!-- Asynchronous HTTP support with Servlet 3.0  -->
-   <dependency>
-      <groupId>org.jboss.resteasy</groupId>
-      <artifactId>async-http-servlet-3.0</artifactId>
-      <version>3.1.3-SNAPSHOT</version>
-   </dependency>
+     It is recommended to declare the BOM in your POM file, that way you will always
+     be sure to get the correct version of the artifacts.  In addition, you will not need
+     to declare the version of each RESTeasy artifact called out in the dependencies section.
+   </para>
+   <para>
 
-</dependencies>
-]]>
-</programlisting>
-    <para>
-        There is also a pom that can be imported so the versions of the individual modules do not have to be specified. Note that maven 2.0.9 is required for this.
-    </para>
-    <programlisting>
-<![CDATA[
+     Declare the BOM file in the dependencyManagement section of the POM file like this.
+     Note that maven version 2.0.9 or higher is required to process BOM files.
+   </para>
+  <programlisting>
+    <![CDATA[
     <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.jboss.resteasy</groupId>
-                <artifactId>resteasy-bom</artifactId>
-                <version>3.1.3-SNAPSHOT</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
+      <dependencies>
+        <dependency>
+          <groupId>org.jboss.resteasy</groupId>
+          <artifactId>resteasy-bom</artifactId>
+          <version>${version.resteasy}</version>
+          <type>pom</type>
+          <scope>import</scope>
+        </dependency>
+      </dependencies>
     </dependencyManagement>
-]]>
-    </programlisting>
+  ]]>
+  </programlisting>
+   <para>
+     Declare the specific RESTeasy artifacts you require in the dependencies section of the
+     POM file like this.
+   </para>
+  <programlisting>
+    <![CDATA[
+     <dependencies>
+       <dependency>
+          <groupId>org.jboss.resteasy</groupId>
+          <artifactId>resteasy-client</artifactId>
+       </dependency>
+        ....
+   </dependencies>
+  ]]>
+  </programlisting>
+   <para>
+     It is possible to reference an RESTeasy artifact version not in the current BOM
+     by specifying a version in the dependency itself.
+   </para>
+  <programlisting>
+    <![CDATA[
+     <dependencies>
+       <dependency>
+          <groupId>org.jboss.resteasy</groupId>
+          <artifactId>resteasy-client</artifactId>
+          <version>${version.some.other}<version>
+       </dependency>
+        ....
+   </dependencies>
+  ]]>
+  </programlisting>
+   <para>
+     Using the BOM file is not required.  Individual RESTEasy artifacts can be
+     declared in the dependencies section of the POM file like this.
+   </para>
+  <programlisting>

```

Comment:
We should not be promoting this approach, right? I'd probably skip this part, it's implicit in the sections above imho. WDYT?